### PR TITLE
Replacing all Unicode icon

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: src/controllers/favorites.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ define(["appRouter", "cardBuilder", "dom", "globalize", "connectionManager", "ap
...
+            section.showTitle || (sectionClass += " verticalSection-extrabottompadding"), html += '<div class="' + sectionClass + ' hide">', html += '<div class="sectionTitleContainer sectionTitleContainer-cards padded-left">', layoutManager.tv ? html += '<h2 class="sectionTitle sectionTitle-cards">' + globalize.translate(section.name) + "</h2>" : (html += '<a is="emby-linkbutton" href="' + getRouteUrl(section, apiClient.serverId()) + '" class="more button-flat button-flat-mini sectionTitleTextButton">', html += '<h2 class="sectionTitle sectionTitle-cards">', html += globalize.translate(section.name), html += "</h2>", html += '<i class="md-icon">chevron_right</i>', html += "</a>"), html += "</div>", html += '<div is="emby-scroller" class="padded-top-focusscale padded-bottom-focusscale" data-mousewheel="false" data-centerfocus="true"><div is="emby-itemscontainer" class="itemsContainer scrollSlider focuscontainer-x padded-left padded-right" data-monitor="markfavorite"></div></div>', html += "</div>"
```
Comment (JustAMan) [inline src/controllers/favorites.js]: can we de-uglify this?

Comment (grafixeyehero) [reply src/controllers/favorites.js]: it already de-uglify in this PR https://github.com/jellyfin/jellyfin-web/pull/548

File: src/components/alphapicker/alphapicker.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ define(['focusManager', 'layoutManager', 'dom', 'css!./style.css', 'paper-icon-b
         html += '<div class="' + rowClassName + '">';
         if (options.mode === 'keyboard') {
             // space_bar icon
-            html += '<button data-value=" " is="paper-icon-button-light" class="' + alphaPickerButtonClassName + '"><i class="md-icon alphaPickerButtonIcon">&#xE256;</i></button>';
+            html += '<button data-value=" " is="paper-icon-button-light" class="' + alphaPickerButtonClassName + '"><i class="md-icon alphaPickerButtonIcon">space_bar</i></button>';
```
Comment (dkanada) [inline src/components/alphapicker/alphapicker.js]: comment here can be removed

File: src/components/alphapicker/alphapicker.js [hunk 3]
```diff
@@ -81,7 +81,7 @@ define(['focusManager', 'layoutManager', 'dom', 'css!./style.css', 'paper-icon-b
 
         if (options.mode === 'keyboard') {
             // backspace icon
-            html += '<button data-value="backspace" is="paper-icon-button-light" class="' + alphaPickerButtonClassName + '"><i class="md-icon alphaPickerButtonIcon">&#xE14A;</i></button>';
+            html += '<button data-value="backspace" is="paper-icon-button-light" class="' + alphaPickerButtonClassName + '"><i class="md-icon alphaPickerButtonIcon">backspace</i></button>';
```
Comment (dkanada) [inline src/components/alphapicker/alphapicker.js]: same here

File: src/components/accessschedule/accessschedule.template.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
 <div class="formDialogHeader">
-    <button is="paper-icon-button-light" class="btnCancel autoSize" tabindex="-1"><i class="md-icon">&#xE5C4;</i></button>
+    <button is="paper-icon-button-light" class="btnCancel autoSize" tabindex="-1">
+        <i class="md-icon">arrow_back</i></button>
```
Comment (dkanada) [inline src/components/accessschedule/accessschedule.template.html]:
```suggestion
        <i class="md-icon">arrow_back</i>
    </button>
```

Comment (grafixeyehero) [reply src/components/accessschedule/accessschedule.template.html]: https://github.com/jellyfin/jellyfin-web/pull/582/commits/bf6b621337d8c8092be530b02eec478aabe9124d

**Pull request merged**